plugins/aea-cli-ipfs/aea_cli_ipfs/ipfs_utils.py

```diff
@@ -57,9 +58,22 @@ def _check_ipfs() -> None:
                 "Please ensure you have version 0.6.0 of IPFS daemon installed."
             )
 
+    @staticmethod
+    def is_started_externally() -> bool:
```

Comment:
The url should be overridable via kwargs imo.

Reply:
Done in 37a0992